test/python/sdpa/frost/test_sdpa_fwd_split_kv_sm100.py

```
@@ -0,0 +1,792 @@
+    mod = load_template(_os.path.join(kdir, kfile), params, tag=f"t_{kfile[:16]}_{splits}_{cta_mma}")
+    fn = mod.compile(b=B, qh=H, kh=H, sq=SQ, skv=SKV, has_lse=True)
+    stream = cuda_driver.CUstream(torch.cuda.current_stream().cuda_stream)
+    o_p = torch.zeros(splits * B, SQ, H, D, device=dev, dtype=torch.float16)
+    lse_p = torch.zeros(splits * B, H, SQ, device=dev, dtype=torch.float32)
+    zH = torch.zeros(H, dtype=torch.float32, device=dev)
+    zB = torch.zeros(B, dtype=torch.int32, device=dev)
+    ps = (B, H, H, SQ, SKV, 0)
+    log2e = cutlass.Float32(scale * _math.log2(_math.e))
+
+    if not mx:
+        mk = lambda *sh: (torch.randn(*sh, device=dev) * 0.5).clamp(-448, 448).to(torch.float8_e4m3fn)
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `mk` as a `def`

(E731)